setup.py

```diff
@@ -29,13 +29,13 @@ def find_resource_files(directory, relative_to=None):
 
 setup(
     name=release_package,
-    version='0.3.1',
-    description='Tethys Application Warehouse',
+    version='1.0.0',
```

Comment:
Oh, i see you're tracking version numbers here. syncing with the install.yml would be a good idea but i don't care that much.